### ISSUE TITLE
The `sockaddr.Include` and `sockaddr.Exclude` do not have "private" selector.

### ISSUE THREAD

**Issue by kwilczynski** [hairyhenderson/gomplate#328]
Hi @hairyhenderson,

I've noticed that the documentation mentions that the `sockaddr.Exclude` and `sockaddr.Include` support a "private" selector, but it does not seem to be the case, for instance:

```
$ gomplate -i '{{ sockaddr.GetAllInterfaces | sockaddr.Include "private" "" }}'
Error: template: <arg>:1:39: executing "<arg>" at <sockaddr.Include>: error calling Include: invalid include selector "private"
```

Which is correct as per the [ifaddrs.go](https://github.com/hashicorp/go-sockaddr/blob/6d291a969b86c4b633730bfc6b8b9d64c3aafed9/ifaddrs.go#L963-L994) implementation.

To achieve this (somewhat), one would have to use the `rfc` selector with an appropriate numeric RFC code. Alternatively, we could send a Pull Request against `go-sockaddr` to add this include/exclude filter.

Would you like to send a Pull Request to update documentation?

**Comment by kwilczynski** [comment]
I have opened a Pull Request at https://github.com/hashicorp/go-sockaddr/pull/23 to add support for `private` to the `include` and `exclude` filters.

**Comment by hairyhenderson** [comment]
Hi @kwilczynski! Thanks for logging this issue. I'm really not sure why I documented `private` in there - probably a copy/paste error from `Sort`.

If you'd like to issue a PR to fix the doc, that'd be great!

From what I can see, doing `sockaddr.Include "rfc" "6890"` should accomplish what you're looking for. With `Sort`, `private` is really just a wrapper around the RFC 6890 comparator.

**Comment by kwilczynski** [comment]
Hi @hairyhenderson,

No worries! Fix to the documentation coming right up!

**Comment by hairyhenderson** [comment]
Fixed in #329
